deploy/helm/sumologic/conf/events/events.conf

```diff
@@ -50,6 +50,8 @@
     @id sumologic.endpoint.events
     sumo_client {{ include "sumologic.sumo_client" . | quote }}
     endpoint "#{ENV['SUMO_ENDPOINT_DEFAULT_EVENTS_SOURCE']}"
+    source_name {{ .Values.fluentd.events.sourceName }}
+    source_category {{ .Values.fluentd.events.source_category | default (printf "%s/%s"  (include "sumologic.clusterNameReplaceSpaceWithDash" .) (.Values.fluentd.events.sourceName ))}}
```

Comment:
Usage of `sumologic.clusterNameReplaceSpaceWithDash` I believe should be documented somewhere, shouldn't it?

Reply:
It should, and we use it basically everywhere we need the cluster name for something. There's even a warning from setup for it I believe. I've added a comment about it to the README and all the other docs explicitly mentioning it.